src/ctia/schemas/search_agg.clj

```diff
@@ -6,15 +6,22 @@
   "Date range query, includes lowerfrom and excludes to"
   {s/Keyword
    (st/optional-keys
-    {:gte s/Inst
-     :lt s/Inst})})
+    {:gte (s/conditional
+            string? s/Str
+            :else s/Inst)
+     :lt (s/conditional
+           string? s/Str
+           :else s/Inst)})})
```

Comment:
Changes in this file were needed after adding schema fixtures to `ctia.stores.es.crud-test` (also in this PR).

Reply:
Will do.